stream_alert_cli/terraform_generate.py

```diff
@@ -280,6 +281,56 @@ def generate_stream_alert(cluster_name, cluster_dict, config):
 
     return True
 
+def generate_cloudwatch_log_metrics(cluster_name, cluster_dict, config):
+    """Add the CloudWatch Metric Filters module to the Terraform cluster dict.
+
+    Args:
+        cluster_name (str): The name of the currently generating cluster
+        cluster_dict (defaultdict): The dict containing all Terraform config for a given cluster.
+        config (dict): The loaded config from the 'conf/' directory
+
+    Returns:
+        bool: Result of applying the cloudwatch metric filters to the stream_alert module
```

Comment:
The function always returns True right now. Is that intended? Maybe you can clarify this

Reply:
@austinbyers @jacknagz  this is to conform to the same precedent set by Jack in all of the `generate...` functions. A good amount of them only return True, and I recall jack wanting to do it for consistency. Can you clarify @jacknagz